zachang/HelloBooks
# #153927218 users should be able to change password

#### What does this PR do?
Implements change password feature.

#### Description of Task to be completed?
It makes it possible for users to change their password on the app.

#### How should this be manually tested?
+ Users should login then click the ` change password` link to navigate a  change password page.

#### What are the relevant pivotal tracker stories?
`#153927218`

#### Questions:
None
